Создан класс App:Application()

manualcheg opened the pull request:
1. Создан класс
2. Добавлена инициализация Koin
3. В файл gradle(app) добавлена зависимость для Koin
4. Добавлен параметр в файл-манифест

manualcheg committed:
1. Создан класс
2. Добавлена инициализация Koin
3. В файл gradle(app) добавлена зависимость для koin

JudjinGM commented on app/src/main/java/ru/practicum/android/diploma/App.kt:
```diff
@@ -0,0 +1,17 @@
+            modules()
+        }
+    }
+}
```
можно добавить в манифест файл app

manualcheg replied:
Нужно, а не можно

manualcheg replied:
Добавил

JudjinGM reviewed (commented):
Добавил предложение

manualcheg committed:
1. Создан класс
2. Добавлена инициализация Koin
3. В файл gradle(app) добавлена зависимость для koin

anonim-no committed:
Merge branch 'dev' into feature/TECH-4/Создание_класса_App
